src/main/Extensions/WindowsControlPanel/WindowsControlPanelItem.ts

```diff
@@ -0,0 +1,6 @@
+export interface WindowsControlPanelItem {
+    Name: string;
+    CanonicalName: string;
+    Description: string;
+    IconBase64: string;
+}
```

Comment:
Please use types instead of interfaces as long as there is no behavior.

```typescript
export type WindowsControlPanelItem = {
    Name: string;
    CanonicalName: string;
    Description: string;
    IconBase64: string;
};
```

Reply:
Interesting, I didn't know types. Why do you prefer this over interfaces?